### ISSUE TITLE
Filter layers on replicate

### ISSUE THREAD

**Issue by hildogjr** [MitjaNemec/ReplicateLayout#67]
Feature request: disable some layers of replicate procedure.

Use case: I usually use a lot of Users.Drawings and Comments in the first layout that attempt to replicate. I could disable the draws of replicate but sometimes some copper / silk draws have to make part of replication.

Propose:

- Back end: create a filter in the replicate function;
- Front end: add a tab to the GUI with this "advanced configuration" with a listbox of available layers into the area to be replicated.

I am user version 2.1.3 with the last correction for KiCad 7.99, not release to automatic update.

**Comment by timoscs** [comment]
My 1 cent here too, I came to suggest this very feature!

**Comment by MitjaNemec** [comment]
Yeah, while the idea is great it is unlikely that it will be implemented.

Lately I just don't seem to find the time and/or the energy to tinker with the plugins. Coupled with the fact that my work shifted so I don't do the PCB layouts as much as I did so I get less internal push to do it.

You might use the "group" feature by putting only the item you wish to replicate into a group.
